app/views/categories/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<section class="login">
+  <%= form_with model: @categories, url:categories_path, scope: :category, method: :post, data: { turbo: false } do |form| %>
+    <nav class="nav">
+      <%= link_to categories_path do %>
+        <i class="fa-solid fa-arrow-left"></i>
+      <% end %>
+      <div>
+        ADD A NEW CATEGORY
+      </div>
+      <div class="actions">
+        <%= form.submit "ADD", class: "loginBtn" %>
+      </div>
+    </nav>
+  
+
+    <div class="textField">
+      <div class="field">
+        <%= form.text_field :name, placeholder: "Category name", autofocus: true, autocomplete: "name", class: "form-control stuff" %>
+      </div>
+
+      <div class="field">
+        <%= form.file_field :icon, placeholder: "Icon", class: "form-control stuff" %>
+      </div>
```

Comment:
- [ ] I would recommend you present the user with their errors when creating a category.

- [ ] **[OPTIONAL]** It would be nice to add the `required: true` attribute here as well for better UX.